CIPs/cip-124.md

```diff
@@ -63,10 +64,15 @@ Where:
 * `0xce` is the streamid multicode as defined in the [multicodec table](https://github.com/multiformats/multicodec/blob/master/table.csv). 
 * `0x05` is the streamtype code for an cip-124 EventID as defined in the [streamTypes table](../tables/streamtypes.csv).
 * `network_id` is a number as defined in the [networkIds table](../tables/networkIds.csv)
-* `sort_value` is based on a user provided value for *sort-key* and *sort-value*
+* `separator_key` is the header field that has the separator_value in it.
+* `separator_value` is the value in the field specified by the separator_key
+* `context_key` is the header field that has the context_value in it.
+* `context_value` is the first 8 bytes of the value in the field specified by the context_key.
+* `blockNumber` is the ETH blockNumber from the after header or a prev time event.
```

Comment:
"after header" isn't really defined yet (CIP is pending) does it make sense to mention it here? Maybe we need to merge the after CIP before this PR?